src/components/vr_cooperation/include/vr_cooperation/commands/base_command_request.h

```diff
@@ -51,7 +51,7 @@ namespace commands {
  * @brief Base command class for requests
  */
 class BaseCommandRequest : public Command,
-  public event_engine::EventObserver<application_manager::MessagePtr, std::string> {
+  public event_engine::EventObserver<application_manager::MessagePtr, int> {
  public:
```

Comment:
You should create separate BaseCommandRequest for HMI. 
Maybe RequestFromHMI.
This BaseCommandRequest use for Mbile request.